addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2861,6 +2861,104 @@ options.registry.topMenuColor = options.Class.extend({
     },
 });
 
+options.registry.BreadcrumbOptions = VisibilityPageOptionUpdate.extend({
+    pageOptionName: "breadcrumb_visible",
+    showOptionWidgetName: "regular_breadcrumb_visibility_opt",
+
+    async visibility(previewMode, widgetValue, params) {
+        await this._super(...arguments);
+        await this._changeVisibility(widgetValue);
+        const targetWindow = this.$target[0].ownerDocument.defaultView;
+        targetWindow.dispatchEvent(new targetWindow.Event("resize"));
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    async _changeVisibility(widgetValue) {
+        if (!(widgetValue !== "hidden")) {
+            return;
+        }
+        const transparent = (widgetValue === "transparent");
+        await new Promise((resolve, reject) => {
+            this.trigger_up("action_demand", {
+                actionName: "toggle_page_option",
+                params: [{name: "breadcrumb_overlay", value: transparent}],
+                onSuccess: () => resolve(),
+                onFailure: reject,
+            });
+        });
+    },
```

Comment:
The header option "over the content" seems to do a lot more while this is basically the same behavior desired. Why is that?